charts/consul/templates/client-daemonset.yaml

```
@@ -503,7 +503,7 @@ spec:
           - mountPath: /consul/login
             name: consul-data
             readOnly: false
-          {{- if (and (not .Values.global.secretsBackend.vault.enabled) (not .Values.externalServers.useSystemRoots)) }}
+          {{- if (and .Values.global.tls.enabled (and (not .Values.global.secretsBackend.vault.enabled) (not .Values.externalServers.useSystemRoots))) }}
```

you can chain multiple cases under an and, ie `and 1 2 3` will and all 3 cases.
```suggestion
          {{- if (and .Values.global.tls.enabled (not .Values.global.secretsBackend.vault.enabled) (not .Values.externalServers.useSystemRoots)) }}
```

---

Ah nice! thanks!